src/sonic-host-services/scripts/hostcfgd

```diff
@@ -758,136 +803,124 @@ class KdumpCfg(object):
             memory = self.kdump_defaults["memory"]
             if data.get("memory") is not None:
                 memory = data.get("memory")
-            if isLoad or data.get("memory") is not None:
+            if data.get("memory") is not None:
                 run_cmd("sonic-kdump-config --memory " + memory)
 
             # Num dumps
             num_dumps = self.kdump_defaults["num_dumps"]
             if data.get("num_dumps") is not None:
                 num_dumps = data.get("num_dumps")
-            if isLoad or data.get("num_dumps") is not None:
+            if data.get("num_dumps") is not None:
                 run_cmd("sonic-kdump-config --num_dumps " + num_dumps)
 
 class NtpCfg(object):
-    def __init__(self, CfgDb):
-        self.config_db = CfgDb
+    """
+    NtpCfg Config Daemon
+    1) ntp-config.service handles the configuration updates and then starts ntp.service
+    2) Both of them start after all the feature services start
+    3) Purpose of this daemon is to propagate runtime config changes in
+       NTP, NTP_SERVER and LOOPBACK_INTERFACE
```

Comment:
Again, This isn't any new functionality. I've just added the comments to show what was happening and modify the method to suit the new design.

Reply:
I've updated the description, if you think, a more detailed explanation is required, i can add that. 

And sure, please attach the owners to review. As for KDumpcfg, the changes are pretty simple. 

As for KDumpCfg, i can explain the changes. Previously, the load() method used to call the kdump_update() which used to do the heavy lifting on the switch. 

All i've did was to remove the invocation of kdump_update() from load() because unlike in the previous case, kdump_update() will at least be called once (given the table is populated), thanks to SubscriberStateTable. So the idea is to avoid an extra call of kdump_update